tests/integration.rs

```diff
@@ -260,6 +260,50 @@ fn test_show_folder_of_symlink_for_long_multi() {
         .stdout(predicate::str::contains("link/:"));
 }
 
+#[test]
+fn test_version_sort() {
```

Comment:
I think `sort.rs` would be a better place for this test than in `integration.rs`.
I guess we will keep this test here anyways, should help with an integration test for sort. But could you have a similar test in `sort.rs`?

Reply:
Maybe also add one with items like this:
```
0.3.7  0.11.5
```
Probably not necessary, but just to be safe.

Reply:
added, IMO, `integration.rs` is where we did the app functional testing, `lsd` as the entry.

of cause, some more package unit tests would always be better.